app/admin_statistics.py

```diff
@@ -25,7 +27,16 @@ class Handler(utils.BaseHandler):
     def get(self):
         repos = sorted(model.Repo.list())
         all_usage = [self.get_repo_usage(repo) for repo in repos]
-        self.render('admin_statistics.html', all_usage=all_usage)
+        note_status_list = []
+        for note_status in NOTE_STATUS_TEXT:
+            if note_status == '':
+                continue
+            note_status_list.append(note_status)
```

Comment:
Optional: You can prepend "num_notes_" here instead of in the template. Then it will also prepend "num_notes" to the "id" attribute, but I believe it's fine.